examples/benchmarks/tablebench/src/test/table.bench.ts

```diff
@@ -126,18 +126,43 @@ describe("Table", () => {
 				);
 			}
 
-			const dataBytes = measureEncodedLength(JSON.stringify(transposeTable(data)));
+			const rowMajorJsonBytes = measureEncodedLength(JSON.stringify(data));
+			const colMajorJsonBytes = measureEncodedLength(JSON.stringify(transposeTable(data)));
```

Comment:
Reports the SharedTree's summary sizes vs. two baselines:

Row-major is the de facto standard from databases like Oracle, PostgreSQL, SQL Server, etc.
Col-major is what some REST APIs use to compact the size.